k8s-deploy/pkg/service/kube_node.go

```diff
@@ -15,12 +15,14 @@
 
 package service
 
+import "github.com/pkg/errors"
+
 // GetNodeIP get a node ip
 func GetNodeIP() ([]string, error) {
 
 	svcs, err := KubeClient.GetNodes("")
 	if err != nil {
-		return nil, err
```

Comment:
We'd better use errors.Wrap() function to keep the original error in the new error.